lib/message.ts

```diff
@@ -272,7 +272,9 @@ export class Message {
   }
 
   isDeadDrop() {
-    return this.getChatId() === C.DC_CHAT_ID_DEADDROP
+    // TODO: Fix
+    //return this.getChatId() === C.DC_CHAT_ID_DEADDROP
+    return false
   }
```

Comment:
the deaddrop system was changed, so that function doesn't exist anymore, see the core changelong: https://github.com/deltachat/deltachat-core-rust/blob/master/CHANGELOG.md#1570

Reply:
I know, but other api is needed i guess